packages/@okta/vuepress-site/docs/api/resources/zones/index.md

```diff
@@ -2,23 +2,160 @@
+
+| Field Name     | Description                                                                                 | Data Type                                     | Required        | Max Length    |
+| :------------- | :------------------------------------------------------------------------------------------ | :-------------------------------------------- | :-------------- | :------------ |
+| gateways       | IP addresses (range or CIDR form) of this zone                                              | Array of [Address Objects](#address-object)   | No              | 150 (entries) |
+| proxies        | IP addresses (range or CIDR form) allowed to forward request from gateway addresses above   | Array of [Address Objects](#address-object)   | No              | 150 (entries) |
+
+#### Address Object
+
+Each address object specifies a set of IP addresses, expressed using either range or CIDR form.
+
+| Field Name  | Description                                                | Data Type   | Required |
+| :---------- | :--------------------------------------------------------- | :---------- | :------- |
+| type        | Format of the value: either CIDR or RANGE                 | String      | Yes       |
+| value       | Value in CIDR/RANGE form depending on the type specified   | String      | Yes       |
+
+#### Address Object Example (CIDR)
+```json
+{
+    "type": "CIDR",
+    "value": "1.2.3.4/24"
+  }
+```
+
+#### Address Object Example (range)
+```json
+{
+    "type": "RANGE",
+    "value": "1.2.3.0-1.2.3.255"
+  }
+```
+
+### IP Zone Example
+```json
+{
+  "type": "IP",
+  "id": "nzouagptWUz5DlLfM0g3",
+  "name": "newNetworkZone",
+  "status": "ACTIVE",
+  "created": "2017-01-24T19:52:34.000Z",
+  "lastUpdated": "2017-01-24T19:52:34.000Z",
+  "system": false,
+  "gateways": [
+    {
+      "type": "CIDR",
+      "value": "1.2.3.4/24"
+    },
+    {
+      "type": "RANGE",
+      "value": "2.3.4.5-2.3.4.15"
+    }
+  ],
+  "proxies": [
+    {
+      "type": "CIDR",
+      "value": "2.2.3.4/24"
+    },
+    {
+      "type": "RANGE",
+      "value": "3.3.4.5-3.3.4.15"
+    }
+  ]
+}
+```
+
+### Dynamic Zone Properties
+
+The follow attributes are defined by Dynamic Zone objects:
+
+| Field Name     | Description                                                                                 | Data Type                                     | Required        | Max Length    |
+| :------------- | :------------------------------------------------------------------------------------------ | :-------------------------------------------- | :-------------- | :------------ |
+| proxyType       | One of the following: `"" or null` (When not specified), `Any`(Meaning any proxy), `Tor`, `NotTorAnonymizer`                                            | String   | No              | N/A |
+| locations        | The locations of this zone   | Array of [Location Objects](#location-object)   | No              | 75 (entries) |
```

Comment:
`The locations of this zone ` -> `The geo locations of this zone`

Reply:
I'll just add that `geolocation` is one word.